.github/workflows/ci-dhge-latex.yaml

```diff
@@ -32,13 +33,34 @@ jobs:
           working_directory: build/
           root_file: example.tex
 
+      - name: Rename example.tex
+        run: mv -v build/example.tex    build/kusche.tex
+
+      - name: Set Kusche-Mode and Abstract
+        run: |
+          sed -i 's/\def\\CKUSCHE *{0}/\def\\CKUSCHE {1}/g' config.tex
+          sed -i 's/\def\\CHASABSTRACT *{0}/\def\\CHASABSTRACT {1}/g' config.tex
+
+      - name: Compile *.tex in Kusche mode
+        uses: xu-cheng/latex-action@v2
+        with:
+          working_directory: build/
+          root_file: kusche.tex
+
       - name: Upload artifact
         if: github.event_name == 'push' && github.ref == 'refs/heads/main'
         uses: actions/upload-artifact@v2
         with:
           name: example
           path: ./build/example.pdf
 
+      - name: Upload Kusche mode artifact
+        if: github.event_name == 'push' && github.ref == 'refs/heads/main'
+        uses: actions/upload-artifact@v2
+        with:
+          name: kusche
+          path: ./build/kusche.pdf
+

```

Comment:
Ich würde es glaube etwas schöner finden, wenn beide PDFs als ein Artefakt hochgeladen werden (siehe beispielsweise folgendes Bild)
![grafik](https://user-images.githubusercontent.com/36958734/143895814-d48daed2-9ba3-4cb9-b167-364a0a336bfb.png)

Reply:
Ist mehr eine Kleinigkeit. Siehe z.B. hier unter [diesem](https://github.com/ZeroPointMax/dhge-latex/actions/runs/1516596666) Action Run stehen die Artifacts. Du kannst diese auch in ein einzelnes artifact verpacken, aber wie gesagt keine Ahnung ob das so viel besser waere. Wuerde davon mal gerne die Meinung von @RvNovae hoeren ansonsten passt alles ja :D

wie immer vielen dank^^

Reply:
oh achso, sorry, bin da nicht wirklich Experte, das können wir machen, ja :D

Aber da du Oliver schonmal geadded hast, können wir auch seine Meinung noch abwarten xD